soroban-spec/src/gen/json.rs

```diff
@@ -50,10 +51,12 @@ pub fn generate_from_wasm(wasm: &[u8]) -> Result<String, FromWasmError> {
 }
 
 pub fn generate(spec: &[ScSpecEntry]) -> String {
-    spec.iter()
-        .map(Entry::from)
-        .map(|e| serde_json::to_string_pretty(&e).expect("serialization of the spec entries should not have any failure cases as all keys are strings and the serialize implementations are derived"))
-        .collect()
+    format!("[{}]",
```

Comment:
Could you change building the JSON manually here, to doing a collect into a Vec<_>, then calling serde_json::to_string_pretty on that vec? It will have us continue to rely on the serde_json crate for serializing and formatting all the JSON consistently. This change as it is results in unformatted.

Alternatively, if you check the box on the PR that lets maintainers push commits I'll push the change and merge it.